code/datums/martial/plasma_fist.dm

```diff
@@ -52,9 +61,50 @@
 	D.visible_message("<span class='danger'>[A] hits [D] with THE PLASMA FIST TECHNIQUE!</span>", \
 					"<span class='userdanger'>You're suddenly hit with THE PLASMA FIST TECHNIQUE by [A]!</span>", "<span class='hear'>You hear a sickening sound of flesh hitting flesh!</span>", null, A)
 	to_chat(A, "<span class='danger'>You hit [D] with THE PLASMA FIST TECHNIQUE!</span>")
+	var/turf/Dturf = get_turf(D)
 	D.gib()
 	log_combat(A, D, "gibbed (Plasma Fist)")
-	return
+	if(nobomb)
+		return
+	if(!isliving(target) || !target.mind)
+		to_chat(A, "<span class='warning'>You cannot power up your </span><span class='notice'>Apotheosis</span><span class='warning'> off of this!</span>")
+		return
+	else if(plasma_power >= plasma_cap)
+		to_chat(A, "<span class='warning'>You cannot power up your </span><span class='notice'>Apotheosis</span><span class='warning'> any more!</span>")
+		new /obj/effect/temp_visual/plasma_soul(Dturf)//doesn't beam to you, so it just hangs around and poofs.
+	else
+		plasma_power++
```

Comment:
Add a var you increase this by so badmins can have their fun, its admin granted after all. Default 1.